### ISSUE TITLE
Add Homebrew Formula

### ISSUE THREAD

**Issue by Kjens93** [mac-cain13/R.swift#453]
At our company, we don't use CocoaPods for dependency resolution/management, but we really like R.swift and it would be really cool to have a simple command to install R.swift as a command-line tool on our development and CI machines, namely:
```bash
brew install rswift
```
It shouldn't be too difficult to create Homebrew Formulae for future releases and start shipping them off to homebrew/homebrew-core as another step in the R.swift build lifecycle.

Information about building Homebrew formulae can be found [here](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md).

**Comment by mac-cain13** [comment]
There actually is a Homebrew formula exactly for this purpose: https://formulae.brew.sh/formula/rswift

@tomasharkema created this formula he didn't publish the 5.0.0.alpha releases yet, but I hope as soon as we have a stable (and fixed the last Xcode 10 issues) he will update this formula!

**Comment by Kjens93** [comment]
@mac-cain13 that's great news! Any way we can get that added to the README? I'm sure I'm not the only developer who's wondered about it.

**Comment by Kjens93** [comment]
I updated the homebrew formula with a `--devel` option (pointed at 5.0.0.alpha.2) and a `-HEAD` option. You can install v5.0.0.alpha.2 by running `brew install --devel rswift`.

**Comment by tomasharkema** [comment]
@Kjens93 thanks for doing that! 

Because I mostly use Cocoapods, I always totally forget to keep the formula up to date. 

@mac-cain13 what would be the right way to go forward? Would be nice to somehow integrate homebrew version bump in your release process, so we don't forget.

**Comment by mac-cain13** [comment]
Yeah, fine by me to automate the Homebrew release also in fastlane. But would be good to have this in a separate lane so it will not affect the normal release proces if it breaks.
